README.md

```diff
@@ -69,7 +71,7 @@ extern crate alloc_system;
 
 * Build the project (in release mode to optimize the size of the binary)
 ```
-cargo build --release
+cargo build --release --target=mipsel-unknown-linux-gnu
```

Comment:
Yeah, mine required it. Guess it doesn't hurt to be specific when cross-compiling.